scala-core-fp/src/test/scala/com/baeldung/scala/functor/FunctorUse.scala

```diff
@@ -0,0 +1,32 @@
+package com.baeldung.scala.functor
+
+import cats.Functor
+import cats.instances.option._
+import org.scalatest.flatspec.AnyFlatSpec
+
+class FunctorSpec extends AnyFlatSpec {
```

Comment:
Need to apply the Scala standard two-space indent.

Reply:
Done